### DISCUSSION TITLE
Mapping of API's with the user journey

### DISCUSSION THREAD

**Discussion by RahulSatanure** [ONDC-Official/ONDC-FIS-Specifications#12] [Q&A] [answered]
Please guide us for how to map given API's with the user journey as the mapping was not provide. If API wise mapping done along with user journey will be great help for API integration.

**Comment by Antriksh-Parmar** [comment]
@92shreyansh

**Comment by 92shreyansh** [comment, marked as answer]
Hi @RahulSatanure the current user journey is illustartated using the flow UI would refine it more will add sequence diagrams to the same for better understanding in a day or two.

**Comment by RahulSatanure** [comment]
@92shreyansh can we get codebase access on Github for this integration inline with other ONDC retail integration.

**Reply by RahulSatanure** [reply]
Please provide clarity on Swagger UI .

Who will use provider platform API's  ( Buyer APP or Lenders ) & who will use consumer API's ( Buyer APP or Lender).

As a lender we need to built the our API's, please guide for the same. accordingly we will start the mapping.

If we get an idea which API's to be build by the lenders then we will start working on the same

**Reply by 92shreyansh** [reply]
Will update the swagger description to clarify this.

**Reply by 92shreyansh** [reply]
Unclear on the previous ask as the current github repo and others are public and everything is available to all

**Reply by RahulSatanure** [reply]
@92shreyansh Thanks for the update . 

In flow API's forms are mentioned but in Swagger API's forms are not mentioned.  can you please clarify the same.

**Reply by 92shreyansh** [reply]
The Form are not part of the core beckn specification transaction APIs and we wanted the swagger to be there for all APIs and not the forms as Forms would vary usecase by use case. 
You can find the illastrative forms as part of the github repo inside api/components/examples/forms

**Comment by RahulSatanure** [comment]
Hi @92shreyansh ,

Can you please provide clarity on the Form's

In the last session you have mentioned that Form's has to be hosted by Lenders platform and once information is received, Lender has to provide submission ID.

ONDC has provided the standardize form's but please let us know, will buyer app will call multiple form's from different lenders which are onboarded on ONDC platform and user has to feel up multiple form of different lenders or if buyer app is rendering the form how data will get to the Lender Platform from "which API".

Please clear my understanding here.

In request body of Onsearch API , Provider Platform has to display the static Loan offer and on the basis of static offer, Buyer App will call select API by choosing any particular static offer. Once the offer is selected by Borrower of a particular lender then the further user journey will be done with that particular lender or borrower can select Multiple offers from and continue journey with multiple lender's ..?

**Reply by RahulSatanure** [reply]
Hi @92shreyansh , Awaiting for your reply

**Reply by madhus1025** [reply]
@92shreyansh  Please reply
